src/commandsAndMenu.tsx

```diff
@@ -1620,13 +1620,28 @@ export async function showGitOperationDialog<T>(
         errorMessage => (error as Error).message.indexOf(errorMessage) > -1
       )
     ) {
+      const remoteURI = await model.getRemotes().then(remote => {
+        return remote;
+      });
+
+      // Determine the Git Provider
+      const re = /https:\/\/(.+)\.com/;
+      const result = remoteURI[0]?.url.match(re);
+      const gitProvider = result[1];
```

Comment:
Result may be undefined so you should protect the code against that; something like:

```suggestion
      const result = remoteURI[0]?.url.match(re) ?? [];
      const gitProvider = result[1];
```